scss/styles.scss

```diff
@@ -1,3 +1,19 @@
-.title {
-    color: red;
-}
\ No newline at end of file
+@import 'colors';
+@import 'sizes';
+
+.content-title {
+	display: flex;
+	align-items: center;
+	position: relative;
+
+	.logo-title {
+		width: 100px;
+	}
+	
+	.title {
+		color: black;
+		font-size: $text-big;
+		position: absolute;
+		left: 78px;
```

Comment:
I would suggest to suggest using multiples of 5 when defining sizes. There is very little difference between using 75/80 instead of 78